project.css

```diff
@@ -29,6 +32,65 @@ html {
     box-shadow: inset 0px 4px 4px rgba(0, 0, 0, 0.25);
 }
 
+.display-of-products {
+    width: 100%;
+    --display-height: 100px;
+    height: var(--display-height);
+    background: white;
+    display: flex;
+    flex-flow: row;
+    justify-content: space-around;
+    z-index: -1;
+    object-fit: contain;
+}
+
+.above {
+    margin-top: 3%;
+    animation: left-to-right 20s linear infinite;
+}
+
+.above2 {
```

Comment:
Why is this named `above2`? 😅

Reply:
I have 2 divs in each row of products so I just named the second one 2 😅